### ISSUE TITLE
Support of parallel environments with multiprocessing for MineDojo

### ISSUE THREAD

**Issue by bmazoure** [MineDojo/MineDojo#96]
Based on similar threads under the minerl and malmo repositories, there are issues with running multiple parallel copies of the minedojo environment, eg via the Gym `SyncVec` or `AsyncVec` interface. From what I understand, these issues stem from running gradle from the same directory for compilation. I've tried to set the `GRADLE_USER_HOME` and `GRADLE_HOME` directories to point to different folders, without success. Are there any tips to making this work reliably?

**Comment by AsWali** [comment]
Hey, Did you manage to figure this out ?

**Comment by bmazoure** [comment]
No, still no progress. It seems like retrying to rebuild the MineDojo env multiple times will (stochastically) work sometimes, but still no reliable fix.
